conf/layer.conf

```diff
@@ -0,0 +1,15 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+	${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "labgrid"
+BBFILE_PATTERN_labgrid = "^${LAYERDIR}/"
+BBFILE_PRIORITY_labgrid = "8"
+
+LAYERDEPENDS_labgrid = "core"
+LAYERDEPENDS_labgrid += "meta-python"
+
+LAYERSERIES_COMPAT_labgrid = "sumo rocko thud"
```

Comment:
In the follow-up PR, you depend on 'snappy' which comes first in with the 'thud' branch of meta-oe.

Thus I fear you are not compatible with sumo and rocko!? Did you test that?

Reply:
@ejoerns The snappy recipe is contained in the same PR, so I'll merge this as is.